docs/technology_stack.adoc

```diff
@@ -7,35 +7,43 @@
 toc::[]
 
 == Overview
-The following sections describe the technology stack used by Planner.
+The Planner is an Angular app built on top of Node.js. It can run as a standalone app, or be used as a library to be integrated into the link:./fabric8-ui[Fabric8 UI].
 
-== Angular 4
-<TBD>
+Development or deployment of Planner is fairly platform agnostic, so we can skip the operating system requirements and discuss from the runtime environment onward.
 
-== CSS and SASS
-<WIP, to be changed to PF CSS, Less>
+== Development Runtime
+Planner uses *`Node.js`* as the platform-runtime, on top of which it builds; and use *`npm`* for resolving and distributing packages for the project.
```

Comment:
use *`npm`*> uses *`npm`* 
We could put it this way too, 
Planner uses:
- `Node.js`* as the platform-runtime, on top of which it builds (the runtime component?)
- *`npm`* for resolving and distributing packages for the project